packages/base/room.gts

```diff
@@ -168,25 +157,11 @@ class ScrollIntoView extends Modifier {
 class EmbeddedMessageField extends Component<typeof MessageField> {
   // TODO need to add the message specific CSS here
   <template>
-    <BoxelMessage
```

Comment:
I had to remove the `BoxelMessage` component from here because the new `AiAssistant::Message` is a host component and can not be used here. I think we should be updating the `BoxelMessage` component with the changes from `AiAssistant::Message` and use it in here instead of duplicating the Message components.